auth_supplier/tests/test_auth_supplier.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.tests.common import TransactionCase
+
+
+class TestSAuthSupplier(TransactionCase):
+    def test_user_signup(self):
```

Comment:
As you see in Travis, this test must start by enabling uninvited users to sign up. Check https://github.com/odoo/odoo/blob/52484b0132ee3e0063dd67cc3c2eb434f99f5db8/addons/auth_signup/res_config.py#L24 for ideas.